Tools/Scripts/run-leaks

```diff
@@ -130,12 +130,6 @@ sub runLeaks($$)
     # of the tool versions where it's supported and needed).
     my @leaksOutput = `/usr/bin/leaks \"$target\" --list`;
 
-    # FIXME: Remove the fallback once macOS Mojave is the oldest supported version.
```

Comment:
Some of these removals seems correct, like this one.

I would recommend breaking removals out into smaller PRs where relevant ones are grouped together. That will make getting reviews easier (it's going to be a challenge finding a single reviewer with sufficient expertise to review all of these changes) and, in the event one of deletions needs to be reverted, the others deletions will be able to remain in the tree.